### PR TITLE
New option to disable default filters

### PR DESCRIPTION
**Pull request by skorn**
Allow setting options to disable the built in regexp filters.

### CODE REVIEW

File: src/swapper.rs [hunk 1]
```diff
@@ -169,6 +169,10 @@ impl<'a> Swapper<'a> {
             return vec!["--regexp".to_string(), format!("'{}'", value)];
           }
 
+          if name.starts_with("disable_default") {
+            return vec!["--disable_default".to_string(), format!("'{}'", value)];
```
Comment (fcsonline) [inline src/swapper.rs:173]: Is `disable_default` or `disable-default`?

**Comment by fcsonline** [comment]
I think this feature is interesting. What about removing the word `default` from all parameters and leave it as `--disable`. It looks like more ergonomic and achieves the same purpose.

**Comment by skorn** [comment]
I had that initially, but wasn’t sure if it was clear enough. I like shorter though so I’m good with disable. 

The hyphen vs underscore was because it didn’t work hyphen didn’t work for me. So for that bit of internal handling changing to underscore worked. But was under the hood so all user interaction is hyphenated. However shortening to disable would remove that obfuscation anyways.

**Comment by fcsonline** [comment]
Should we push this?

**Pull request closed without merging**